scss/editor/_theme.scss

```diff
@@ -1,12 +1,54 @@
 @include exports("editor/theme") {
+    $editor-overlay-background: #000 !default;
+    $editor-resize-handle-border: transparent transparent $widget-border !default;
 
     .k-editor {
         @include appearance( header );
 
-
         .k-editable-area {
             @include appearance( input );
         }
+
+        .k-resize-handle {
+            border-color: $editor-resize-handle-border;
+        }
+    }
+
+    .k-overlay {
+        background-color: $editor-overlay-background;
+    }
+
+
+    .k-header.k-window-content.k-content {
+        @include appearance( header );
     }
 
+    .k-editor .k-editor-toolbar-wrap,
+    .k-ct-popup .k-editor-toolbar,
+    .k-header.k-window-content.k-content {
```

Comment:
`.k-header.k-window-content.k-content` does not say much. Perhaps we need a more descriptive class, or to drop some of the selectors?